### DISCUSSION TITLE
SV pipeline for local installation

### DISCUSSION THREAD

**Discussion by cogrho** [broadinstitute/seqr#3168] [General]
Hi!

Is it possible to load structural variants or at least CNV variants from GATK gCNV files to seqr local instance? If not, is it planned to support SV and CNV calls for local installs at some point? 

Thanks!

**Comment by hanars** [comment]
CNV and SV calls are currently not supported in local seqr, but we do have plans to add that support in 2023. The major barrier right now is that the CNV/SV call files we used for our original implementation are a bit "bespoke" and do not conform to any standard/ publicly available calling pipeline. Our hope is to standardize our pipeline this year to be able to either handle standard GATK files and/or provide clear documentation about how to generate the input files seqr requires

**Reply by cogrho** [reply]
Thank you for the explanation! SV pipeline would be a great addition to the existing local pipeline.

**Reply by wengkhong** [reply]
I also have a local seqr instance (which works great by the way... thanks!) and echo the sentiment that having a way to ingest CNV + SV calls would be great. We're more than happy to reformat our call files to whatever input file format seqr requires.

**Reply by hanars** [reply]
I'm glad to hear about the interest! This is a high priority for us to get done this year, it just unfortunately has some barriers that make it a slower process than we would like

**Reply by jxchong** [reply]
We don't have a local instance (ours is in the cloud) but we were able to get GATK-SV calls into Seqr though there's reformatting necessary until they finish building official support for the current version of GATK-SV.

See posts by @karynne7 here for snippets:
https://github.com/broadinstitute/seqr/issues/3064

**Reply by anjalijain22** [reply]
Hi @hanars! I was wondering if there was any update on the SV pipeline for local instances? Like everyone mentioned, we also have a local instance of seqr and it would be great if we could also add in our SV/CNV calls to seqr. If there is any help you need from a user perspective, I'd be happy to support it. Thanks very much!

**Reply by hanars** [reply]
We are unfortunately still blocked on having a stable, publicly available calling pipeline for this data. We are hoping to have that finalized in late fall of this year

**Reply by anjalijain22** [reply]
That sounds good! Thanks for your prompt response!

**Reply by anjalijain22** [reply]
Hi Hana! Hope you're well :) 
Apologies to keep bugging you but I was wondering if there was any update on the CNV/SV pipeline for local installs? We have a ton of positive feedback from our users of the seqr local instance for small variants and there is a growing interest to use seqr for SV/CNV data exploration as well. Thanks in advance for your help!

**Reply by hanars** [reply]
Hi Anjali, unfortunately not. We will post in the announcements channel once there is any progress on this front! We are glad folks are finding seqr so useful and we are excited to make this functionality available as soon as we can get over the technical and data-normalization hurdles

**Reply by anjalijain22** [reply]
Thanks for sharing this, we're are looking forward to it!

**Comment by ghost** [comment]
@tranthn 
@erickarlins
@shmenolin

**Comment by bnouyou** [comment]
Hello,

I am wondering if it's possible now to import SV data in seqr (local instance) ?
Thank you.

**Reply by hanars** [reply]
Hello, as answered above there is unfortunately no support for structural variation in local installations at this time

**Comment by YasirKusay** [comment]
Hi, are there any updates for this?

**Reply by hanars** [reply]
I am sorry but as answered above there is unfortunately no support for structural variation in local installations at this time. We will post a very clear announcement once support has been added
